tests/test_coils.py

```diff
@@ -254,6 +325,142 @@ def test_dunder_methods(self):
         assert coils1[-2][0].__class__ is coil1.__class__
 
 
+@pytest.mark.unit
+def test_save_and_load_MAKEGRID_coils(tmpdir_factory):
+    """Test loading in and saving CoilSets from MAKEGRID format files."""
+    Ncoils = 22
+    input_path = f"./tests/inputs/coils.MAKEGRID_format_{Ncoils}_coils"
+    tmpdir = tmpdir_factory.mktemp("coil_files")
+    tmp_path = tmpdir.join(f"coils.MAKEGRID_format_{Ncoils}_coils")
+    shutil.copyfile(input_path, tmp_path)
+
+    coilset = CoilSet.from_makegrid_coilfile(str(tmp_path))
+    assert len(coilset) == Ncoils  # correct number of coils
+    # TODO: add better tests for this?
```

Comment:
Did you want to add better tests in this PR?

Reply:
yep, done